pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -245,15 +276,16 @@ func (c *Reconciler) reconcile(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 		// or it has been torn down.
 	} else if err != nil {
 		logger.Errorf("Error checking Endpoints %q: %v", pa.Spec.ServiceName, err)
```

Comment:
This logline should move to the `err` check after `FetchReadyAddressCount` is called.

Reply:
Done.